lib/dns.js

```diff
@@ -175,12 +175,12 @@ function lookupService(host, port, callback) {
 
   var req = new GetNameInfoReqWrap();
   req.callback = callback;
-  req.host = host;
+  req.hostname = hostname;
```

Comment:
Just to understand this: isn't this name used later on during the resolution on the libuv side? And if it's not then what's the use of assigning it here?
/cc @addaleax @jasnell

Reply:
@lundibundi I think the only use we currently have for it is `this.hostname` in `onlookupservice()` above? I might be missing something, though.

Reply:
Looks like it as I haven't found anything down to the `libuv/unix/getnameinfo`, though I may have missed it. Though it looks kind of strange that we pass hostname both via `req`/`this` and as an argument in case of success.

Well, anyway, CI is almost green so looks like this is fine.